### ISSUE TITLE
you should port a fnf mod 

### ISSUE THREAD

**Issue by LEVIXDDLMAO** [Dominicthecodingenuis/Fnaf2_SCRATCH#1]
can i tell you what mod?

**Comment by Dominicthecodingenuis** [comment]
yep

**Comment by LEVIXDDLMAO** [comment]
thx and let work with each other
So the mod https://github.com/ArcyDev/Vs-Retrospecter

**Comment by Dominicthecodingenuis** [comment]
on it

**Comment by LEVIXDDLMAO** [comment]
you wanna work with each other?

**Comment by LEVIXDDLMAO** [comment]
make it html5

**Comment by LEVIXDDLMAO** [comment]
e

**Comment by LEVIXDDLMAO** [comment]
homie wheres the html5 port https://github.com/ArcyDev/Vs-Retrospecter

**Comment by Dominicthecodingenuis** [comment]
bro the fricking thing had a error

**Comment by LEVIXDDLMAO** [comment]
you want there newer version??

**Comment by LEVIXDDLMAO** [comment]
nevermind what was the error??

**Comment by Dominicthecodingenuis** [comment]
source/DisclaimerState.hx:109: characters 9-18 : Type not found : FileCache
source/FreeplayState.hx:684: characters 68-69 : Missing ;
source/FreeplayState.hx:690: characters 80-81 : Missing ;
source/Character.hx:43: characters 15-24 : Type not found : FileCache
source/Character.hx:66: characters 15-24 : Type not found : FileCache
source/Character.hx:88: characters 15-24 : Type not found : FileCache
source/Character.hx:112: characters 15-24 : Type not found : FileCache
source/Character.hx:127: characters 15-24 : Type not found : FileCache
source/Character.hx:142: characters 15-24 : Type not found : FileCache
source/Character.hx:155: characters 15-24 : Type not found : FileCache
source/Character.hx:168: characters 15-24 : Type not found : FileCache
source/Character.hx:181: characters 15-24 : Type not found : FileCache
source/Character.hx:194: characters 15-24 : Type not found : FileCache
source/Character.hx:207: characters 15-24 : Type not found : FileCache
source/Character.hx:220: characters 15-24 : Type not found : FileCache
source/Character.hx:233: characters 15-24 : Type not found : FileCache
source/Character.hx:246: characters 15-24 : Type not found : FileCache
source/Character.hx:289: characters 15-24 : Type not found : FileCache
source/Character.hx:336: characters 15-24 : Type not found : FileCache
source/Character.hx:356: characters 15-24 : Type not found : FileCache
source/Character.hx:370: characters 15-24 : Type not found : FileCache
source/Character.hx:384: characters 15-24 : Type not found : FileCache
source/Character.hx:398: characters 15-24 : Type not found : FileCache
source/Character.hx:412: characters 15-24 : Type not found : FileCache
source/Character.hx:426: characters 15-24 : Type not found : FileCache
source/Character.hx:441: characters 15-24 : Type not found : FileCache
source/Character.hx:476: characters 15-24 : Type not found : FileCache
source/Character.hx:522: characters 15-24 : Type not found : FileCache
source/Character.hx:568: characters 15-24 : Type not found : FileCache
source/Character.hx:613: characters 15-24 : Type not found : FileCache
source/Character.hx:662: characters 15-24 : Type not found : FileCache
source/Character.hx:680: characters 15-24 : Type not found : FileCache
source/Character.hx:700: characters 15-24 : Type not found : FileCache
source/Note.hx:120: characters 18-27 : Type not found : FileCache
source/Note.hx:147: characters 19-28 : Type not found : FileCache
source/Note.hx:151: characters 19-28 : Type not found : FileCache
source/Note.hx:189: characters 15-24 : Type not found : FileCache
source/Note.hx:250: characters 17-26 : Type not found : FileCache
source/Note.hx:261: characters 17-26 : Type not found : FileCache
source/Note.hx:314: characters 18-27 : Type not found : FileCache
source/Note.hx:318: characters 18-27 : Type not found : FileCache
source/PlayState.hx:571: characters 25-34 : Type not found : FileCache
source/PlayState.hx:617: characters 22-31 : Type not found : FileCache
source/PlayState.hx:640: characters 28-37 : Type not found : FileCache
source/PlayState.hx:663: characters 22-31 : Type not found : FileCache
source/PlayState.hx:686: characters 28-37 : Type not found : FileCache
source/PlayState.hx:731: characters 26-35 : Type not found : FileCache
source/PlayState.hx:761: characters 32-41 : Type not found : FileCache
source/PlayState.hx:787: characters 27-36 : Type not found : FileCache
source/PlayState.hx:816: characters 23-32 : Type not found : FileCache
source/PlayState.hx:851: characters 24-33 : Type not found : FileCache
source/PlayState.hx:899: characters 27-36 : Type not found : FileCache
source/PlayState.hx:921: characters 24-33 : Type not found : FileCache
source/PlayState.hx:947: characters 31-40 : Type not found : FileCache
source/PlayState.hx:971: characters 24-33 : Type not found : FileCache
source/PlayState.hx:992: characters 23-32 : Type not found : FileCache
source/PlayState.hx:1016: characters 29-38 : Type not found : FileCache
source/PlayState.hx:1254: characters 31-40 : Type not found : FileCache
source/PlayState.hx:1282: characters 23-32 : Type not found : FileCache
source/PlayState.hx:1289: characters 7-16 : Type not found : FileCache
source/PlayState.hx:2473: characters 27-36 : Type not found : FileCache
source/PlayState.hx:2486: characters 28-37 : Type not found : FileCache
source/PlayState.hx:2490: characters 28-37 : Type not found : FileCache
source/PlayState.hx:2545: characters 27-36 : Type not found : FileCache
source/PlayState.hx:2558: characters 28-37 : Type not found : FileCache
source/PlayState.hx:2562: characters 28-37 : Type not found : FileCache
source/PlayState.hx:2629: characters 27-36 : Type not found : FileCache
source/PlayState.hx:2658: characters 29-38 : Type not found : FileCache
source/PlayState.hx:2689: characters 35-44 : Type not found : FileCache
source/Options.hx:121: characters 38-47 : Type not found : FileCache
source/Options.hx:148: characters 40-49 : Type not found : FileCache
source/GameplayCustomizeState.hx:91: characters 18-27 : Type not found : FileCache
source/GameplayCustomizeState.hx:105: characters 18-27 : Type not found : FileCache
source/GameplayCustomizeState.hx:131: characters 20-29 : Type not found : FileCache
source/GameplayCustomizeState.hx:146: characters 19-28 : Type not found : FileCache
source/PauseSubState.hx:313: characters 3-17 : You cannot access the sys package while targeting js (for sys.FileSystem)

**Comment by LEVIXDDLMAO** [comment]
do ou wanna do a diff mod?

**Comment by Dominicthecodingenuis** [comment]
yeah

**Comment by LEVIXDDLMAO** [comment]
okay let me find one

**Comment by LEVIXDDLMAO** [comment]
try this one https://github.com/aflacc/vsaflac-kade

**Comment by Dominicthecodingenuis** [comment]
it worked

**Comment by LEVIXDDLMAO** [comment]
or this one https://github.com/ShadowMario/NewVeryCoolPsychic

**Comment by LEVIXDDLMAO** [comment]
okay thats good if you can you port this one too? 
https://github.com/ShadowMario/NewVeryCoolPsychic you dont have to

**Comment by LEVIXDDLMAO** [comment]
are you gonna upload it?

**Comment by Dominicthecodingenuis** [comment]
yeah

**Comment by LEVIXDDLMAO** [comment]
where is it? :0

**Comment by LEVIXDDLMAO** [comment]
can you port this? https://github.com/ShadowMario/NewVeryCoolPsychic  to html5 and wheres my free robux and vbucks >:0 / im talking baout your site

**Comment by Dominicthecodingenuis** [comment]
ok

**Comment by Dominicthecodingenuis** [comment]
i did not promise free vbucks and robux im not kreekcraft :)

**Comment by LEVIXDDLMAO** [comment]
lmao bro when i saw this i died of laughter

**Comment by Dominicthecodingenuis** [comment]
its not working

**Comment by LEVIXDDLMAO** [comment]
whats the error?

**Comment by LEVIXDDLMAO** [comment]
oh and what haxe do you have installed? if you do use haxe

**Comment by Dominicthecodingenuis** [comment]
haxe 4.2.5

**Comment by Dominicthecodingenuis** [comment]
i deleted the file but it was this source/PlayState.hx:1345: characters 8-16 : FlxVideo has no field disabled source/PlayState.hx:1361: characters 25-33 : FlxVideo has no field disabled source/PlayState.hx:1362: characters 25-34 : FlxVideo has no field vlcBitmap source/PlayState.hx:1363: characters 25-34 : FlxVideo has no field vlcBitmap

**Comment by LEVIXDDLMAO** [comment]
may be update it?

**Comment by LEVIXDDLMAO** [comment]
if you cab

**Comment by Dominicthecodingenuis** [comment]
its at the max update level

**Comment by Dominicthecodingenuis** [comment]
for rn

**Comment by LEVIXDDLMAO** [comment]
ah

**Comment by LEVIXDDLMAO** [comment]
bro

**Comment by LEVIXDDLMAO** [comment]
Yo you wanna make a fnf engine with me?

**Comment by Dominicthecodingenuis** [comment]
yeah but i cant work on it now my mom said i couldn't use my laptop because i failed i one of my grades

**Comment by LEVIXDDLMAO** [comment]
nah bro your fine i wanna work on it later we would need a mod first with the engine

**Comment by LEVIXDDLMAO** [comment]
So i wanna make a charater selectior for the engine im slowly adding shit there if im need email me here Dontblock012@outlook.com
